pydantic/__init__.py

```diff
@@ -161,5 +164,20 @@
     'VERSION',
 ]
 
+# A mapping of {<member name>: <module name>} defining dynamic imports
+_dynamic_imports = {'RootModel': '.root_model'}
+if typing.TYPE_CHECKING:
+    from .root_model import *
```

Comment:
```suggestion
    from .root_model import RootModel
```
Any reason that we shouldn't do this? I'm okay with the `import *` if necessary just seems better since you can cmd+f etc.

Reply:
This is consistent with the approach we use for other non-dynamic imports. However, for dynamics imports we still need to define each one explicitly in `_dynamic_imports`. Thus, we can use explicit import here as using the wildcard import doesn't simplify the workflow of adding a new dynamic import much.

Go to definition is working as expected and I don't see a profit in allowing this line popping up in the search results for "RootModel" as the class is defined elsewhere.

IMO, the wildcard here follows the way static imports are being defined and makes an explicit reference to them by making it visually similar.

Reply:
I think an explicit `from .root_model import RootModel` might help IDEs and static type checkers more, though I can't prove that for sure.